src/server_manager/web_app/app.ts

```diff
@@ -437,7 +437,8 @@ export class App {
       // Wait for server config to load, then update the server view and list.
       if (isManagedServer(server)) {
         try {
-          for await (const _ of server.monitorInstallProgress()) {}
+          // eslint-disable-next-line @typescript-eslint/no-unused-vars
+          for await (const _ of server.monitorInstallProgress()) {/* empty */}
```

Comment:
i know it's not your code but wouldn't `await Promise.all(server.monitorInstallProgress())` work, here? I'm a bit confused. don't have to change haha

Reply:
No, it's an async for loop with an async generator. You need to iterate.

Reply:
ah, it's a generator